### PR TITLE
4772 adds test cases

### LINKED ISSUE #4772: missing cuda test cases
**Describe the bug**
```python
Python 3.8.0 (default, Nov  6 2019, 21:49:08) 
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from tests.utils import TEST_NDARRAYS
>>> print(TEST_NDARRAYS)
(<built-in function array>, <built-in method as_tensor of type object at 0x7f53e1f3b9c0>, functools.partial(<class 'monai.data.meta_tensor.MetaTensor'>, meta={'a': 'b', 'affine': tensor([[2., 0., 0., 0.],
        [0., 2., 0., 0.],
        [0., 0., 2., 0.],
        [0., 0., 0., 1.]])}))
>>> import torch
>>> print(torch.cuda.is_available())
True
>>> 
```
`TEST_NDARRAYS` should include the cuda-base test cases when cuda is available.

### PR DESCRIPTION
**Pull request by wyli**
Signed-off-by: Wenqi Li <wenqil@nvidia.com>

Fixes #4772

the updated version in the PR will have the cuda test cases:
```py
>>> from tests.utils import TEST_NDARRAYS
>>> print(TEST_NDARRAYS)
(<built-in function array>, <built-in method as_tensor of type object at 0x7f9fe45bf9c0>, functools.partial(<built-in method as_tensor of type object at 0x7f9fe45bf9c0>, device='cuda'), functools.partial(<class 'monai.data.meta_tensor.MetaTensor'>, meta={'a': 'b', 'affine': tensor([[2., 0., 0., 0.],
        [0., 2., 0., 0.],
        [0., 0., 2., 0.],
        [0., 0., 0., 1.]])}))
>>> 
```

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: tests/utils.py [hunk 1]
```diff
@@ -736,10 +736,10 @@ def test_local_inversion(invertible_xform, to_invert, im, dict_key=None):
     assert_allclose(im_inv.affine, im_ref.affine, atol=1e-3, rtol=1e-3)
 
 
-TEST_TORCH_TENSORS: Tuple[Callable] = (torch.as_tensor,)
+TEST_TORCH_TENSORS: Tuple = (torch.as_tensor,)
 if torch.cuda.is_available():
     gpu_tensor: Callable = partial(torch.as_tensor, device="cuda")
-    TEST_NDARRAYS = TEST_TORCH_TENSORS + (gpu_tensor,)
+    TEST_TORCH_TENSORS = TEST_TORCH_TENSORS + (gpu_tensor,)
```
Comment (Nic-Ma) [inline tests/utils.py:742]: I see the issue, so actually here we define `TEST_NDARRAYS` and re-define later without GPU..
https://github.com/Project-MONAI/MONAI/blob/dev/tests/utils.py#L742

Thanks.

**Review by Nic-Ma — APPROVED** [review]

**Review by KumoLiu — APPROVED** [review]

**Pull request merged**